packages/core/data-transfer/types/providers.d.ts

```diff
@@ -19,6 +19,8 @@ export interface ISourceProvider extends IProvider {
   streamLinks?(): NodeJS.ReadableStream | Promise<NodeJS.ReadableStream>;
   streamMedia?(): NodeJS.ReadableStream | Promise<NodeJS.ReadableStream>;
   streamConfiguration?(): NodeJS.ReadableStream | Promise<NodeJS.ReadableStream>;
+  getSchemas?(): any;
```

Comment:
We probably don't need to expose this one, it could just be private in `LocalStrapiSourceProvider`

Reply:
Although on second thought, maybe we should require schemas to be available synchronously? They should always be reasonably sized, and the schema validation step really shouldn't be waiting for a stream to compare them. Maybe we do want this. Thoughts?

Reply:
we might want to use that for the integration step, comparing both schemas as we don't really need a stream at that step

Reply:
Yes, the goal is to be able to have access to the schemas (which should be of reasonable size) synchronously for the schema validation & integrity checks, and using a stream in order to transfer them if needed using the same mechanism & transform tools that we use for other data types.